app/packages/operators/src/useCustomPanelHooks.ts

```diff
@@ -131,19 +137,7 @@ export function useCustomPanelHooks(props: CustomPanelProps): CustomPanelHooks {
   );
 
   useEffect(() => {
-    if (props.onLoad && !isLoaded) {
-      executeOperator(
-        props.onLoad,
-        { panel_id: panelId },
-        {
-          callback(result) {
-            const { error: onLoadError } = result;
-            setPanelStateLocal((s) => ({ ...s, onLoadError, loaded: true }));
-          },
-        }
-      );
-    }
-
+    onLoad();
```

Comment:
At line 145, should we remove `props.onLoad`, `isLoaded `, and `setPanelStateLocal` and add `onLoad`?